planus/src/errors.rs

```diff
@@ -18,6 +18,7 @@ impl std::error::Error for Error {
 }
 
 #[derive(Copy, Clone, Debug)]
+#[non_exhaustive]
```

Comment:
Don't we want it to be exhaustive at some point?

Reply:
I'm not sure. I think it's safer to leave it as `non_exhaustive` for now, since we can always remove it later.